src/components/Header.test.js

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from '@testing-library/react';
+import Header from './Header';
+import { BrowserRouter } from 'react-router-dom';
+import cat1 from '../assests/cat-4912133_1280.png'
+
+
+
+
+describe("<Header />", () =>{
+  it ("checks the images in header", () => {
+    render(
+      <BrowserRouter>
+        <Header />
+      </BrowserRouter>
+    )
+
+
+    // screen.logTestingPlaygroundURL()
+    const image = screen.getByRole('img', {
+      name: /my\-image/i
+    })
+    expect(image).toHaveAttribute('src', cat1)
+    
+  })
+  
+  // it ("checks my secondary header images with link", () => {
+  //   render(
+  //     <BrowserRouter>
+  //       <Header />
+  //     </BrowserRouter>
+  //   )
+    
+  //   const image2 =  getByRole('img', { name: /my\-image/i })
+  //   expect(image2).to 
+    
+  // ;
+
+  // })
```

Comment:
Is this a test that you want?  If so, please comment back in and make it pass.  If not, please remove it.